src/test/java/com/mentoring/framework/utils/CookieUtils.java

```diff
@@ -0,0 +1,25 @@
+package com.mentoring.framework.utils;
+
+import lombok.extern.slf4j.Slf4j;
+import org.openqa.selenium.Cookie;
+import org.openqa.selenium.WebDriver;
+
+@Slf4j
+public final class CookieUtils {
+
+    private static final String VALIDATION_AND_ANIMATION_DELAY = "0";
+
+    private CookieUtils() {
+    }
+
+    public static void setTestCookies(WebDriver driver) {
+        setCookie(driver, "search-result-timeout", VALIDATION_AND_ANIMATION_DELAY);
+        setCookie(driver, "validation-timeout", VALIDATION_AND_ANIMATION_DELAY);
```

Comment:
I recommend to push the 'name' attributes at least to static final Strings. It would be a better solution to create a new resource file or a final class for the cookie names if there are more than 4-5.

Reply:
@adamveszelinov fixed, pls approve if there is no other items to discuss.